app/src/main/res/values/strings.xml

```diff
@@ -380,4 +380,8 @@
     <string name="date_to">Date To</string>
     <string name="send_sms">Send SMS</string>
     <string name="requested_sms">Requested SMS</string>
+    <string name="enrolled_policies">Enrolled Policies</string>
```

Comment:
Did you added these keys in Lokalize?

Reply:
You mean that these keys should be added in Translations Editor which will automatically add entries in "values-fr/strings.xml" file ?

Reply:
No I didn't. Damian give me a link to policies app in Lokalize - https://app.lokalise.com/project/260745865d22f770125fe4.66491371/ but I don't have a permissions. Could you give me an access to this resosurces? druchniewicz@soldevelo.com

Reply:
You have now access to Lokalize.

Reply:
Don't forget to add the keys in French too.

Reply:
I update the French translation on Lokalize to be synchronised.